locales/it/index.yml

```
@@ -1249,6 +1250,25 @@ wallet:
       expirationDate: mm/aa
       securityCode: "•••"
       securityCode4D: "••••"
+    accessibility:
+      holder:
+        base: "Intestatario carta, inserisci come riportato sulla carta"
+        error: "Intestatario carta. Per favore, non utilizzare lettere accentate"
+      pan:
+        base: "Numero carta"
+        error: "Numero carta. Per favore, inserisci almeno 14 cifre"
```

```suggestion
        error: "Numero carta. Per favore, inserisci almeno {{minLength}} cifre"
```
Could it be more safe get that value [from code](https://github.com/pagopa/io-app/blob/d0f20f4aba3195a506244be1eb657f456a7aca34/ts/utils/input.ts#L7)?

---

Sure, modified in [4f455eb](https://github.com/pagopa/io-app/pull/3020/commits/4f455eb5761527ea415c1c13e525a60384331c7f)